example/src/main/java/uk/gov/moj/cpp/jobmanager/example/BakeryService.java

```diff
@@ -0,0 +1,56 @@
+package uk.gov.moj.cpp.jobmanager.example;
+
+import static java.time.ZonedDateTime.now;
+import static java.util.UUID.randomUUID;
+
+import uk.gov.justice.services.common.converter.ObjectToJsonObjectConverter;
+import uk.gov.moj.cpp.jobstore.api.JobRequest;
+import uk.gov.moj.cpp.jobstore.api.JobService;
+
+import javax.inject.Inject;
+import javax.transaction.Status;
+import javax.transaction.SystemException;
+import javax.transaction.UserTransaction;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class BakeryService {
+
+    protected final Logger logger = LoggerFactory.getLogger(getClass());
+
+    @Inject
+    protected ObjectToJsonObjectConverter objectConverter;
+
+    @Inject
+    private JobService jobService;
+
+    @Inject
+    private UserTransaction userTransaction;
+
+    public void makeCake() {
+
+        final MakeCakeWorkflow firstTask = MakeCakeWorkflow.firstTask();

```

Comment:
Does MakeCakeWorkflow.firstTask() need to be static?

Reply:
I think so otherwise you'd have to reference it like e.g. MakeCakeWorkflow.GET_INGREDIENTS.firstTask()